src/store/modules/fungibleTokens.js

```diff
@@ -85,5 +85,36 @@ export default {
       );
       return commit('setAePublicData', aeternityData);
     },
+    async createOrChangeAllowance(
+      { rootState: { sdk }, state: { selectedToken }, rootGetters: { activeNetwork, account } },
+      amount,
+    ) {
+      const tokenContract = await sdk.getContractInstance(FUNGIBLE_TOKEN_CONTRACT, {
+        contractAddress: selectedToken.contract,
+      });
+      const { decodedResult } = await tokenContract.methods.allowance({
+        from_account: account.publicKey,
+        for_account: activeNetwork.tipContractV2.replace('ct_', 'ak_'),
+      });
+      const allowanceAmount =
+        decodedResult !== undefined
+          ? new BigNumber(decodedResult)
+              .multipliedBy(-1)
+              .plus(convertToken(amount, selectedToken.decimals))
+              .toNumber()
+          : convertToken(amount, selectedToken.decimals).toFixed();
+      return tokenContract.methods[
+        decodedResult !== undefined ? 'change_allowance' : 'create_allowance'
+      ](activeNetwork.tipContractV2.replace('ct_', 'ak_'), allowanceAmount);
+    },
+    async transferAllowance({ rootState: { sdk }, state: { selectedToken } }, [toAccount, amount]) {
```

Comment:
```suggestion
    async transfer({ rootState: { sdk }, state: { selectedToken } }, [toAccount, amount]) {
```

I think the naming there should not be `tranferAllowance` but rather just `transfer` as we are using the `transfer` entrypoint of the contract. 💭